### ISSUE TITLE
Exception through the Python interface.

### ISSUE THREAD

**Issue by vacary** [siconos/siconos#440]
The exceptions that are thrown by the C++ code and handled by the Python inteface, but without any message. This renders very difficult for the user to know where is the exception and how to correct to code.

Is it possible to get the C++ exception message in the python console?

An example :

```python

from numpy.linalg import norm
from siconos.kernel import LagrangianLinearTIDS, NewtonImpactNSL,\
    LagrangianLinearTIR, Interaction, NonSmoothDynamicalSystem, MoreauJeanOSI,\
    TimeDiscretisation, LCP, TimeStepping
from siconos.kernel import SimpleMatrix, getMatrix


from numpy import eye, empty, float64, zeros

t0 = 0       # start time
T = 10       # end time
h = 0.005    # time step
r = 0.1      # ball radius
g = 9.81     # gravity
m = 1        # ball mass
e = 0.9      # restitution coeficient
theta = 0.5  # theta scheme

#
# dynamical system
#
x = [1, 0, 0]    # initial position
v = [0, 0, 0]    # initial velocity
mass = eye(3)  # mass matrix
mass[2, 2] = 2. / 5 * r * r

# the dynamical system
ball = LagrangianLinearTIDS(x, v, mass)

# set external forces
weight = [-m * g, 0, 0]
ball.setFExtPtr(weight)

#
# Interactions
#

# ball-floor
H = [[1, 0, 0, 0]] # Wrong size of the interaction

nslaw = NewtonImpactNSL(e)
relation = LagrangianLinearTIR(H)
inter = Interaction(nslaw, relation)


#
# Model
#
bouncingBall = NonSmoothDynamicalSystem(t0, T)

# add the dynamical system to the non smooth dynamical system
bouncingBall.insertDynamicalSystem(ball)

# link the interaction and the dynamical system
bouncingBall.link(inter, ball)

#
# Simulation
#

# (1) OneStepIntegrators
OSI = MoreauJeanOSI(theta)

# (2) Time discretisation --
t = TimeDiscretisation(t0, h)

# (3) one step non smooth problem
osnspb = LCP()

# (4) Simulation setup with (1) (2) (3)
s = TimeStepping(bouncingBall,t, OSI, osnspb)


# end of model definition

#
# computation
#


# the number of time steps
N = int((T - t0) / h)

# Get the values to be plotted
# ->saved in a matrix dataPlot

dataPlot = zeros((N+1, 5))

#
# numpy pointers on dense Siconos vectors
#
q = ball.q()
v = ball.velocity()
p = ball.p(1)
lambda_ = inter.lambda_(1)


#
# initial data
#
dataPlot[0, 0] = t0
dataPlot[0, 1] = q[0]
dataPlot[0, 2] = v[0]
dataPlot[0, 3] = p[0]
dataPlot[0, 4] = lambda_[0]

k = 1
try:
# time loop
    while s.hasNextEvent():
        s.computeOneStep()

        dataPlot[k, 0] = s.nextTime()
        dataPlot[k, 1] = q[0]
        dataPlot[k, 2] = v[0]
        dataPlot[k, 3] = p[0]
        dataPlot[k, 4] = lambda_[0]
        
        k += 1
        s.nextStep()
except Exception as e :
    print(type(e))
    print('Exception', e)
```

**Comment by fperignon** [comment]
Fixed in 5c566dc

**Comment by vacary** [comment]
Thanks @fperignon . This works like a charm.
